exasol_script_languages_container_tool/lib/tasks/upload/upload_container_base_task.py

```diff
@@ -66,7 +65,7 @@ def generate_command_line_output_str(
         command_line_output_str = textwrap.dedent(
             f"""
             Uploaded {release_path} to
-            {self._get_upload_url(export_info, without_login=True)}
+            {self.build_file_path_in_bucket(export_info).as_udf_path()}
```

Comment:
Haven't found a better way from `exasol.bucketfs` to print target path

Reply:
Because the url is not valid for SaaS. Here the behavior of SaaS and onprem BucketFS differ to much. With that the question is what so want to print here at all. The UDF path gives you the path inside of the UDFs. However, for SaaS it requires quite a lot of knowledge to know that /buckets/uploads/default means it got uploaded to SaaS.

Reply:
Ok, changed the implementation here so that it builds the target URL manually.